app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(players_list: list[Player]) -> int:
```

Comment:
```suggestion
def calculate_team_total_rating(players: list[Player]) -> int:
```
Redundant suffix in all places